nova/virt/vmwareapi/vm_util.py

```diff
@@ -121,6 +121,104 @@ def __init__(self, cpu_limits=None, hw_version=None,
         self.hw_video_ram = hw_video_ram
 
 
+class HistoryCollectorItems(object):
```

Comment:
Can we make this class python3-ready, please? We're going to switch to python3 rather sooner than later (planned train upgrade for april).
Seems like we can just rename `next` to `__next__` and derive from https://six.readthedocs.io/#six.Iterator

Reply:
Done.